### PR TITLE
Feature/test cases

### PR DESCRIPTION
**Pull request by zarmeenmustafa651**

### CODE REVIEW

File: spec/models/appointment_spec.rb [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      @availability = Availability.create(@availability_params)
+      @patient_params = FactoryGirl.attributes_for(:patient)
+      @patient_params[:hospital] = @hospital
+      @patient = Patient.create(@patient_params)
+      @appointment_params = FactoryGirl.attributes_for(:appointment)
+      @appointment_params[:hospital] = @hospital
+      @appointment_params[:doctor] = @doctor
+      @appointment_params[:patient] = @patient
+      @appointment_params[:availability] = @availability
+      @appointment = @hospital.appointments.create(@appointment_params)
```
Comment (MirahSaqib) [inline spec/models/appointment_spec.rb]: code repetition
use one before(:each) block

File: Gemfile [hunk 2]
```diff
@@ -53,11 +53,13 @@ end
...
   gem 'webdrivers'
-  gem 'rspec-rails'
+  gem 'rspec-rails','=4.0.1'
   # To create factories while testing
-  gem 'factory_girl_rails', require: false
+  gem 'factory_girl_rails','=4.9.0', require: false
   # Faker
-  gem 'faker'
+  gem 'faker','=2.14.0'
+  gem 'database_cleaner','=1.8.5'
```
Comment (MirahSaqib) [inline Gemfile]: remove database cleaner

File: spec/rails_helper.rb [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  exit 1
+end
+RSpec.configure do |config|
+  # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
+  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+
+  # If you're not using ActiveRecord, or you'd prefer not to run each of your
+  # examples within a transaction, remove the following line or assign false
+  # instead of true.
+  config.use_transactional_fixtures = false
```
Comment (MirahSaqib) [inline spec/rails_helper.rb]: revert back to true and try running test cases

File: Gemfile [hunk 4]
```diff
@@ -53,11 +53,13 @@ end
 group :test do
```
Comment (MirahSaqib) [inline Gemfile:53]: reverse merge with develop

File: spec/rails_helper.rb (hunk 3 above)
Comment (zarmeenmustafa651) [reply spec/rails_helper.rb]: worked fine with true

**Pull request merged**